README.md

```diff
@@ -167,8 +167,8 @@ The repository includes these files:
 pool:
   vmImage: ubuntu-latest
 steps:
-  - task: InstallMATLAB@0
```

Comment:
We really should remove this section of the README. It often is out-of-date with the actual YAML files.